logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -226,7 +227,7 @@ internal class ConnectionDataSource(
                         teamId = null,
                         protocolInfo = ConversationEntity.ProtocolInfo.Proteus,
                         creatorId = connection.from,
-                        lastNotificationDate = null,
+                        lastNotificationDate = Clock.System.now().toString(),
```

Comment:
Please do not use regular `.toString()`, it omits milliseconds in the string if they are equal to 0, that's why I had to do this change: https://github.com/wireapp/kalium/pull/1258

Reply:
Awesome, thanks. So I'll wait for you RP to get merged and then I'll update my PR.